### PR TITLE
Add `Skin of the Year 2023: Voting Open` news post

### PR DESCRIPTION
**Pull request by 0x84f**
- [x] Remove TBD's from total skins submitted to the compendium once info is available
- [x] Update contest link
- [x] Make sure that we are happy with the 1x version of the banner as the current ver looks a bit crunchy
- [x] Re-date and re-time

Web preview: https://osu3.roanh.dev/home/news/2024-01-14-skin-of-the-year-2023

### CODE REVIEW

File: news/2024/2024-01-01-skin-of-the-year-2023.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+All skins that have been released during 2023 and have been added to [the compendium](https://compendium.skinship.xyz/) are eligible to compete, **including contest & podium entries**. Everyone gets **15** votes each to cast, and you can spend as many or as few as you like.
+
+The top 3 entries will all receive a unique profile badge based on their entry along with 4, 3 and 2 months of osu!supporter respectively. Entries #4 — #10 will all receive a month of osu!supporter each.
+
+This year we had **TBD** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15), **TBD** of them being added to the compendium. We highly suggest you get to some light browsing and check out the submitted skins, and you might find a new skin you will be able to call your "main".
+
+[**Click here to get involved and vote for 2023's Skin of the Year!**](https://osu.ppy.sh/community/contests/170)
+
+The voting will remain open for **14 days** after this post goes up, closing officially on the **TBD**. Hopefully, that should be enough time to go through the catalogue and pick your favourites.
```
Comment (Walavouchey) [inline news/2024/2024-01-01-skin-of-the-year-2023.md:27]: TBD template

**Comment by peppy** [comment]
This isn't waiting on me is it?

**Comment by Walavouchey** [comment]
no, this is waiting on rockroller and co.

File: news/2024/2024-01-01-skin-of-the-year-2023.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Now that we've had a brief glimpse into what's been happening over the past year, let's quickly go over the contest details.
+
+## Voting details
+
+All skins that have been released during 2023 and have been added to [the compendium](https://compendium.skinship.xyz/) are eligible to compete, **including contest and podium entries**. Everyone gets **15** votes each to cast, and you can spend as many or as few as you like.
+
+The top 3 entries will all receive a unique profile badge based on their entry along with 4, 3 and 2 months of osu!supporter respectively. Entries #4–10 will all receive a month of osu!supporter each.
+
+This year we had **TBD** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15), **TBD** of them being added to the compendium. We highly suggest you get to some light browsing and check out the submitted skins, and you might find a new skin you will be able to call your "main".
```
Comment (RockRoller01) [inline news/2024/2024-01-01-skin-of-the-year-2023.md:23]:
```suggestion
This year we had **269** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15), **124** of them being added to the compendium. We highly suggest you get to some light browsing and check out the submitted skins, and you might find a new skin you will be able to call your "main".
```

Final tally is done, here are the numbers

**Review by RockRoller01 — COMMENTED** [review]
As I already mentioned on discord, probably should add imagemaps to year review

**Comment by 0x84f** [comment]
> As I already mentioned on discord, probably should add imagemaps to year review

https://github.com/ppy/osu-wiki/pull/10756/commits/973867f4e297e7684b8755e14eb487550912b633

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**